Makefile

```diff
@@ -67,7 +67,17 @@ vet: ## Run go vet against code.
 
 .PHONY: test
 test: manifests generate fmt vet envtest ## Run tests.
```

Comment:
Is the `test` name still appropriate given it's now only targeting controllers/webhooks? 🤔 
I'm not sure what appropriate naming is best here but I think the nuances of `make test` vs `make unit-test` may cause confusion

Reply:
sure, we can change it, `unit-test` and `component-test` might be good?

Reply:
That sounds good. Maybe `test` can run both?

Reply:
I would rather keep it separated, it's not that big of a deal to run 2 make commands

Reply:
Fine by me

Reply:
adapted